release/src/main/scripts/build_release_candidate.sh

```diff
@@ -275,7 +276,7 @@ if [[ $confirmation = "y" ]]; then
   git clone ${GIT_REPO_URL}
   cd ${BEAM_ROOT_DIR}
   git checkout ${RELEASE_BRANCH}
-  cd sdks/python && tox -e docs
+  cd sdks/python && pip install -r build-requirements.txt && tox -e py37-docs
```

Comment:
this is interesting cause I thought this script uses python2, but `tox -e py37-docs` looks works by python2

Reply:
There are 2 virtualenvs here. The one in line 272 will use the default python version, and the one created by tox will use python3.7.
Tox runs "python setup.py sdist" in the first virtualenv and installs the tarball in the second.

Reply:
Thanks for clarifying this.